### PR TITLE
Fix EC2 fluid storage bus -> AE2FC conversion.

### PR DESCRIPTION
**Pull request by firenoo**
Fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12823

Temporary until we move the transformer over to EC2.

Tested in full pack (2.3.0 RC-3)

### CODE REVIEW

**Comment by Glease** [comment]
FYI [you can turn off spotless for a block of code using comment](https://github.com/diffplug/spotless/blob/main/plugin-gradle/README.md#spotlessoff-and-spotlesson)

**Comment by firenoo** [comment]
Thanks for the tip! Fixed the formatting so it makes better sense as to what it's doing. Basically the way the slots were indexed do not carry over 1:1, so you need to map them using a table or equation (AE2FC indexes by column, while EC2 indexes by row)

File: src/main/java/appeng/parts/CableBusContainer.java [hunk 1]
```diff
@@ -848,6 +862,14 @@ private NBTTagCompound[] covertEC2Part(NBTTagCompound part, NBTTagCompound data)
                 data_new.setInteger("priority", data.getInteger("priority"));
                 data_new.setString("ACCESS", data.getString("access"));
                 NBTTagCompound fluid_filter = new NBTTagCompound();
+                NBTTagList upgrades = data.getTagList("upgradeInventory", TAG_COMPOUND);
+                NBTTagCompound upgradeNew = new NBTTagCompound();
+                upgradeNew.setTag("#0", upgrades.getCompoundTagAt(0));
```
Comment (repo-alt) [inline src/main/java/appeng/parts/CableBusContainer.java:867]: There is just one possible upgrade?

Comment (firenoo) [reply src/main/java/appeng/parts/CableBusContainer.java:867]: Yes, see the screenshot in the linked issue

**Review by repo-alt — APPROVED** [review]

**Pull request merged**